client/app/scripts/liveblog-edit/posts.service.js

```diff
@@ -93,10 +102,24 @@ define([
                 mainItem: post.groups[1].refs[0],
                 items: post.groups[1].refs
             });
-            // complete Post With User Information
-            userList.getUser(post.original_creator).then(function(user) {
-                post.user = user;
+            post.comments = _.reduce(post.groups[1].refs, function(is, val) {
+                return is || _.isUndefined(val.item.name);
+            }, false);
+            // check if `fullDetails` flag is needed
+            // comments items set falg to true.
+            post.fullDetails = post.comments;
+            // special cases for comments.
```

Comment:
I agree with the finish. I'm just talking about the code. I find it strange to have a special case for comments in the posts service because of our view needs. This logic should belongs to the view from my point of view.